modules/client-configuration/pages/system-set-manager.adoc

```diff
@@ -0,0 +1,558 @@
+| Remove Packages           | {check}       | {check}
+| Verify Packages           | {check}       | {cross}
+| Create Groups             | {check}       | {check}
+| Manage Groups             | {check}       | {check}
+| Channel Memberships       | {check}       | {check}
+| Channel Subscriptions     | {check}       | {cross}
+| Deploy/Diff Channels      | {check}       | {cross}
+| Autoinstall Clients       | {check}       | {cross}
+| Tag for Snapshot          | {check}       | {cross}
+| Remote Commands           | {check}       | {cross}
+| Power Management          | {check}       | {cross}
+| Update System Preferences | {check}       | {check}
+| Update Hardware Profiles  | {check}       | {check}
+| Update Package Profiles   | {check}       | {check}
+| Set/Remove Custom Values  | {check}       | {check}
+| Reboot Clients            | {check}       | {check}
+| Migrate clients to another Organization | {check} | {check}
+| Delete clients            | {check} | {check}
+|===
+
+
+You can select clients for the SSM in several ways:
+
+* Navigate to menu:Systems[System List] and check the clients you want to work with.
+* Navigate to menu:Systems[System Groups], and click btn:[Use in SSM] for the system group you want to work with.
+* Navigate to menu:Systems[System Groups], check the group you want to work with, and click btn:[Work with Group].
+
+When you have selected the clients you want to work with, navigate to menu:Systems[System Set Manager], or click the [guimenu]``systems selected`` icon in the top menu bar.
+
+
+[NOTE]
+====
+The details in SSM might differ slightly from the details in other parts of the {productname} {webui}.
+In SSM, all available updates are shown.
+This allows you to upgrade to packages that might not be the latest version.
+====
+
+
+
+== Change Base Channels in SSM
+
+You can use SSM to change the base channel of more than one client at the same time.
+
+[IMPORTANT]
+====
+Changing the base channel significantly changes the packages and patches available to the affected clients.
+Use with caution.
+====
+
+
+
+.Procedure: Using SSM to Change Base Channels for Multiple Clients
+
+. In the {productname} {webui}, navigate to menu:Systems[System List], check the clients you want to work with, and navigate to menu:Systems[System Set Manager].
+. Navigate to the [guimenu]``Channels`` subtab.
+. Locate the current base channel in the list, and verify that the number shown in the [guimenu]``Systems`` column is correct.
+You can click the number in this column to see more details of the clients you are changing.
+. Select the new base channel in the [guimenu]``Desired base Channel`` field, and click btn:[Next].
+. For each child channel, select [guimenu]``No change``, [guimenu]``Subscribe``, or [guimenu]``Unsubscribe``, and click btn:[Next].
+. Check the changes you are making, and choose a time for the action to occur.
+. Click btn:[Confirm] to schedule the changes.
+
+
+
+
+
+////
```

Comment:
I know why you commented this.  Nebvetheless using this technique for more than some 30 or 40 lines (1 page) is a no-op.  I rather vote to put it in a separate file.

Reply:
Most of it will get deleted, yet, but I take your point.